make/images.mk

```diff
@@ -173,6 +173,10 @@ validate-runenv-image: ## Validate runtime image
 	${KM_DOCKER_VOLUME} \
 	${RUNENV_DEMO_IMG_TAGGED} \
 	"$${tmp_bash_array[@]}" | grep "${RUNENV_VALIDATE_EXPECTED}"
+	tmp_bash_array=${RUNENV_VALIDATE_CMD} && \
+	${PODMAN_RUN_TEST} \
+	docker-daemon:${RUNENV_DEMO_IMG_TAGGED} \
```

Comment:
I don't understand the `docker-daemon` here

Reply:
podman has its own local image repository that it will use by default.
We need to tell podman to get the image from the docker daemon's local image repositry.